content/docs/deployment/csmoperator/release/_index.md

```diff
@@ -31,4 +31,39 @@ Description: >
 - [#990 - [BUG]: X_CSI_AUTH_TYPE cannot be set in CSM Operator](https://github.com/dell/csm/issues/990)
 
 ### Known Issues
-There are no known issues in this release.
\ No newline at end of file
+
+| Issue | Workaround |
+|-------|------------|
+| The images of sideCars are currently missing in the sample YAMLs in the offline bundle. As a consequence, the csm-operator is pulling them from registry.k8s.io. | We recommend manually updating the images of sideCars in the sample YAML file, for example, `storage_csm_powerflex_v290.yaml`, before proceeding with the driver installation. Here is an example snippet for the sideCars section in the YAML file:
```

Comment:
Let us change storage_csm_powerflex_v290.yaml to storage_csm_powerflex_v291.yaml

Reply:
fixed